tdmpc2/common/math.py

```diff
@@ -9,19 +9,19 @@ def soft_ce(pred, target, cfg):
 	return -(target * pred).sum(-1, keepdim=True)
 
 
-@torch.jit.script
+
 def log_std(x, low, dif):
 	return low + 0.5 * dif * (torch.tanh(x) + 1)
 
 
-@torch.jit.script
+
 def _gaussian_residual(eps, log_std):
 	return -0.5 * eps.pow(2) - log_std
 
 
-@torch.jit.script
+LOG2PI = 1.8378770351409912
 def _gaussian_logprob(residual):
-	return residual - 0.5 * torch.log(2 * torch.pi)
+	return residual - 0.5 * LOG2PI
```

Comment:
this was breaking in eager (when removing the jit)
Also jit + compile is some uncharted territory. I'm removing them all